### PR TITLE
js kernel can evaluate javascript code

### PR DESCRIPTION
**Pull request by colombod**
html kernel can evaluate javascript in when <script> tags are added and contains code

### CODE REVIEW

File: src/microsoft-dotnet-interactive/src/htmlKernel.ts [hunk 1]
```diff
@@ -45,7 +46,8 @@ export interface HtmlDomFragmentInserterConfiguration {
     getOrCreateContainer?: () => HTMLElement,
     updateContainerContent?: (container: HTMLElement, htmlFragment: string) => void,
     createMutationObserver?: (callback: MutationCallback) => MutationObserver,
-    normalizeHtmlFragment?: (htmlFragment: string) => string
+    normalizeHtmlFragment?: (htmlFragment: string) => string,
+    jsEvaluator?: (js: string) => void
```
Comment (brettfo) [inline src/microsoft-dotnet-interactive/src/htmlKernel.ts:50]: is this ever explicitly set anywhere?

Comment (colombod) [reply src/microsoft-dotnet-interactive/src/htmlKernel.ts:50]: 17 lines later ...

Comment (brettfo) [reply src/microsoft-dotnet-interactive/src/htmlKernel.ts:50]: I meant by specifying it in the config value.  I don't ever see it being specified, so why not leave it as `eval(...)`?

Comment (colombod) [reply src/microsoft-dotnet-interactive/src/htmlKernel.ts]: if done that way is the eval of node during testing and not the one from the browser engine

**Review by brettfo — APPROVED** [review]

**Pull request merged**